packages/bot/src/stores/run-panel-store.js

```diff
@@ -120,6 +121,11 @@ export default class RunPanelStore {
     @action.bound
     toggleDrawer(is_open) {
         this.is_drawer_open = is_open;
+        const el_run_panel  = document.querySelector('.run-panel');
+
+        if (el_run_panel) {
+            scrollWorkspace(Blockly.derivWorkspace, el_run_panel.offsetWidth, true, is_open);
```

Comment:
imo, the workspace shouldn't be scroll when `run-panel` is opened.

Reply:
This is what we discussed previously, but let's talk about it again in our standup.